src/main/play-doc/operation/ConsolidatedLogging.md

```diff
@@ -40,6 +40,28 @@ The provided Elasticsearch bundle configuration is using these settings:
 
 To change the settings create a new bundle configuration by modiying the `bundle.conf` file inside of the bundle configuration zip file. The configuration file is named `elasticsearch-prod-{digest}.zip` and is located in the `extra` folder as well. Afterwards reload the bundle with the new configuration.
 
+### Elasticsearch bundle rolling upgrade
+
+Elasticsearch stores its data within the filesystem. As such, to preserve existing data when performing deployment, it's important to ensure the data is replicated to the new instance of Elasticsearch.
+
+This can be achieved by following these steps.
+
+* Commission a new ConductR node where the new Elasticsearch instance is going to execute.
+* Start this new node and ensure this node is able to successfully join the existing ConductR cluster.
+* Turn off one of the old instance where Elasticsearch is running.
```

Comment:
Operator does not need to know which node ES is running on, as long as after shutting down old node the operator waits for cluster health to turn green.

I think the key thing is operator has to wait long enough for cluster health state to turn green. If the old node doesn't have ES, it will stay green.

If the old node has ES, it will go green -> yellow -> green - but it won't turn yellow at the instant the old node is shutdown. There's a few seconds delay before cluster state turn yellow.